util/reggen/clocking.py

```diff
@@ -39,6 +40,10 @@ def from_raw(raw: object, only_item: bool, where: str) -> 'ClockingItem':
 
         return ClockingItem(clock, reset, primary)
 
+    def set_reg_clock(self, clock: str) -> None:
```

Comment:
What does the `clock` argument do here?

Reply:
i feel like this was code left over when i was trying to do something else here...removing it completely.